README.md

```diff
@@ -84,32 +87,67 @@ git clone https://github.com/Torantulino/Auto-GPT.git
 $ cd 'Auto-GPT'
 ```
 
-3. Install the required dependencies:
+3. Create and activate Virtual Environment, OPTIONAL
+```
+python -m venv ./
+```
+mac + linux:
+```
+source ./bin/activate
+```
+windows:
+```
+./scripts/activate
+```
+4. Install the required dependencies:
+
 *(Again, type this into your CMD window)*
 ```
 pip install -r requirements.txt
 ```
 
-4. Rename `.env.template` to `.env` and fill in your `OPENAI_API_KEY`. If you plan to use Speech Mode, fill in your `ELEVEN_LABS_API_KEY` as well.
+5. Rename `.env.template` to `.env` and fill in your `OPENAI_API_KEY`. If you plan to use Speech Mode, fill in your `ELEVEN_LABS_API_KEY` as well.
   - Obtain your OpenAI API key from: https://platform.openai.com/account/api-keys.
   - Obtain your ElevenLabs API key from: https://elevenlabs.io. You can view your xi-api-key using the "Profile" tab on the website.
   - If you want to use GPT on an Azure instance, set `USE_AZURE` to `True` and provide the `OPENAI_API_BASE`, `OPENAI_API_VERSION` and `OPENAI_DEPLOYMENT_ID` values as explained here: https://pypi.org/project/openai/ in the `Microsoft Azure Endpoints` section
 
 ## 🔧 Usage
 
-1. Run the `main.py` Python script in your terminal:
+1. Run the `autogpt` Python module in your terminal:
 *(Type this into your CMD window)*
 ```
-python scripts/main.py
+<<<<<<< HEAD
+PYTHONPATH=. python autogpt/main.py
+=======
+python -m autogpt
+>>>>>>> autogpt-namespace-fix-imports
```

Comment:
What is intended to be here? Believe this is from the multi pull requests?

Reply:
After mergin in my PR, you now have __main__.py which should replace main.py, but I see that in this PR main.py still exists.

```
autogpt/__init__.py
autogpt/__main__.py
```
This structure makes it possible to use `python -m autogpt`
The __init__.py makes autogpt folder a proper python module.
The __main__.py makes it the default entrypoint for a command line interface program. (See: https://docs.python.org/3/library/__main__.html#main-py-in-python-packages)

I recommend using `python -m autogpt` and removing main.py.

This is the way.

Reply:
> What is intended to be here? Believe this is from the multi pull requests?

Just a mistake, will fix now.

Reply:
@dhensen resolved
@taylor-ennen resolved